icc.xcodeproj/project.pbxproj

```diff
@@ -1767,7 +1767,7 @@
 				LastUpgradeCheck = 0630;
 				TargetAttributes = {
 					3E9F4C4A181F07B700F33B48 = {
-						DevelopmentTeam = V8KL7848Q4;
```

Comment:
Doesn't really matter, but since this isn't a substantive change, probably worth removing from this PR.